src/applications/claims-status/components/AddFilesFormOld.jsx

```diff
@@ -302,22 +309,19 @@ class AddFilesFormOld extends React.Component {
           <a href="/disability/how-to-file-claim">How to File a Claim</a> page.
         </div>
         <div>
-          <va-button
-            primary
```

Comment:
Why are we removing this? I'm assuming because it's redundant to have it?

Reply:
I added the why in my ticket summary.
Primary is no longer a prop on va-button for uswds and and we dont need it.